### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by nil0x42** [andresriancho/w3af#17469]
Hi ! It's been a long time since last release, some commits probably deserve to be more widely adopted by the large majority of people who use Release version from package managers.

**Comment by andresriancho** [comment]
You are completely right. I've neglected this. Will try to release a new version next week.
